src/applications/vaos/tests/services/healthcare-service/index.unit.spec.js

```diff
@@ -128,62 +128,95 @@ describe('VAOS Healthcare service', () => {
       );
       expect(error?.resourceType).to.equal('OperationOutcome');
     });
-  });
 
-  describe('getSupportedHealthcareServicesAndLocations', () => {
-    it('should make successful request', async () => {
-      await getSupportedHealthcareServicesAndLocations({
-        siteId: '983',
-        parentId: '983GC',
-        typeOfCareId: '123',
+    describe('Test makeing call to Vista Scheduling Provider (VSP) api', () => {
```

Comment:
I'm not sure I'm following the test changes in here. It looks like you have two new tests for `getAvailableHealthcareServices` that set `useVSP` to true, which makes sense, but I think you want to put them in the describe section for `getAvailableHealthcareServices` at the top of the file

Reply:
I typically like to have 1 expect statement per test. I know there are some exceptions to this but that's what I'm shooting for. I don't know if there are some best practices that are being followed for writing unit test but I will make the change.

Reply:
Great article. https://medium.com/@me_37286/yoni-goldberg-javascript-nodejs-testing-best-practices-2b98924c9347

Reply:
Having two tests is totally fine with me, I was just suggesting they should be in the same describe group as the other tests for that function.

Reply:
Got it.